src/save_warning_popup.cpp

```diff
@@ -0,0 +1,20 @@
+#include "popups.hpp"
+#include <iostream>
+
+void save_warning_popup(bool &popup, const std::string &msg) {
+
+  ImGui::OpenPopup("Save warning");
```

Comment:
This a generic warning and save warning suggests that you failed to save something or something similiar.
At least i get the text "Save Warning" displayed in the head of the popup if i press the radio button "User current Waveform".
```suggestion
  ImGui::OpenPopup("Warning");
```

Reply:
Agreed.